auth-web/src/components/auth/account-settings/product/ProductPackage.vue

```diff
@@ -115,183 +137,261 @@
+  Account,
+  AccountStatus,
+  Product as ProductEnum,
+  ProductStatus,
+  Role,
+  TaskType
+} from '@/util/constants'
+import {
+  OrgProduct,
+  OrgProductCode,
+  OrgProductsRequestBody
+} from '@/models/Organization'
+import { computed, defineComponent, onMounted, reactive, ref, toRefs } from '@vue/composition-api'
+import CautionBox from '@/components/auth/common/CautionBox.vue'
 import ModalDialog from '@/components/auth/common/ModalDialog.vue'
 import Product from '@/components/auth/common/Product.vue'
+import { ProductStatusMsgContentIF } from '@/models/external'
+import { storeToRefs } from 'pinia'
+import { useAccountChangeHandler } from '@/composables'
 import { useOrgStore } from '@/stores/org'
 import { useUserStore } from '@/stores/user'
+import { userAccessDisplayNames } from '@/resources/QualifiedSupplierAccessResource'
 
-@Component({
+export default defineComponent({
+  name: 'ProductPackage',
   components: {
+    CautionBox,
     Product,
     ModalDialog
-  }
-})
-export default class ProductPackage extends Mixins(AccountChangeMixin) {
-  @State(useOrgStore) public currentOrganization!: Organization
-  @State(useUserStore) public currentUser!: KCUserProfile
-  @State(useOrgStore) public productList!: OrgProduct[]
-  @State(useOrgStore) public currentSelectedProducts!: []
-
-  @Action(useOrgStore) public resetCurrentSelectedProducts!:() => void
+  },
+  setup () {
+    const confirmDialog: InstanceType<typeof ModalDialog> = ref(null)
 
-  @Action(useOrgStore) public getOrgProducts!:(orgId: number) => Promise<OrgProduct>
-  @Action(useOrgStore) public addOrgProducts!:(product:OrgProductsRequestBody) => Promise<OrgProduct>
-  @Action(useOrgStore) public addToCurrentSelectedProducts!:(productCode:any) => Promise<void>
-  @Action(useOrgStore) public syncCurrentAccountFees!:(accoundId:number) => Promise<AccountFee[]>
-  @Action(useOrgStore) public fetchOrgProductFeeCodes!:() => Promise<OrgProductFeeCode>
-  @Action(useOrgStore) public updateAccountFees!:(accountFee) => Promise<any>
+    const { currentUser } = useUserStore()
+    const { setAccountChangedHandler } = useAccountChangeHandler()
+    const {
+      resetCurrentSelectedProducts,
+      getOrgProducts,
+      addOrgProducts,
+      addToCurrentSelectedProducts,
+      syncCurrentAccountFees,
+      fetchOrgProductFeeCodes,
+      updateAccountFees
+    } = useOrgStore()
 
-  public isBtnSaved = false
-  public disableSaveBtn = false
-  public isLoading: boolean = false
-  public isProductActionLoading: boolean = false
-  public dialogTitle = ''
-  public dialogText = ''
-  public dialogIcon = ''
-  public submitRequestValidationError = ''
-  public expandedProductCode: string = ''
-  public AccountEnum = Account
-  public orgProductsFees:any = ''
-  public orgProductFeeCodes:any = ''
-  public isProductActionCompleted: boolean = false
+    const {
+      currentOrganization,
+      productList,
+      currentSelectedProducts
+    } = storeToRefs(useOrgStore())
```

Comment:
A way of preserving reactivity between the composable properties and it's local use here, without redefining each as a local computed value.